BootloaderCommonPkg/Library/IasImageLib/IasImage.c

```diff
@@ -73,25 +77,48 @@ IsIasImageValid (
     return NULL;
   }
 
-  Key.Signature = RSA_KEY_IPP_SIGNATURE;
-  CopyMem (&Key.PubKeyMod, ((UINT8 *) IAS_PUBLIC_KEY (Hdr)), sizeof (Key.PubKeyMod));
+  SignPool = (UINT8 *) AllocatePool(sizeof(IAS_SIGNATURE_INFO) + RSA2048NUMBYTES);
```

Comment:
Instead of using AllocatePool/FreePool et al, can we just declare a UINT buffer the sizeof 256 byte RSA key and RSA signature and use that?

Reply:
Done